test/e2e/app-dir/server-source-maps/server-source-maps-edge.test.ts

```diff
@@ -58,7 +58,7 @@ describe('app-dir - server source maps edge runtime', () => {
 
       const cliOutput = stripAnsi(next.cliOutput.slice(outputIndex))
       expect(cliOutput).toContain(
-        '\n ⨯ Error: ssr-throw' +
+        '\nError: ssr-throw' +
```

Comment:
Do we know why this is changed? Is it not logging through our Logger.error?

Reply:
Wasn't exposing the `routerServerContext` so it was falling back to `console.error` yea although probably better to expose it locally.